src/pages/all-on-one-mini-man/all-on-one-mini-man.ts

```diff
@@ -0,0 +1,105 @@
+import { HazardRecorderProvider } from './../../providers/hazard-recorder/hazard-recorder';
+import { Component } from '@angular/core';
+import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import { FaultsScorecardProvider } from '../../providers/faults-scorecard/faults-scorecard';
+import { FaultStoreProvider } from '../../providers/fault-store/fault-store';
+
+/**
+ * Generated class for the AllOnOnePage page.
+ *
+ * See https://ionicframework.com/docs/components/#navigation for more info on
+ * Ionic pages and navigation.
+ */
+
+@IonicPage()
+@Component({
+  selector: 'page-all-on-one-mini-man',
+  templateUrl: 'all-on-one-mini-man.html',
+})
+export class AllOnOneMiniManPage {
+  isDButtonPressed = false;
```

Comment:
These two variables have very similar names, I've already confused one for the other while doing this review

Reply:
This was copied over too.. not sure I want to go changing this, considering it's just a prototype though.

Reply:
eh it's my abbreviation. I wanted to avoid too long variable names.